plugins/lookup/k8s.py

```diff
@@ -277,7 +264,11 @@ def run(self, terms, variables=None, **kwargs):
         if self.name:
             return [k8s_obj.to_dict()]
 
-        return k8s_obj.to_dict().get('items')
+        items = k8s_obj.to_dict().get('items')
+
+        if len(items) == 1:
```

Comment:
checking the ``len`` means that the ``items`` is already a list, otherwise it will check the size of a ``dict`` or ``string`` ? and if it is already a list we can just return it

Reply:
Good point.